snowflake_utils/__main__.py

```diff
@@ -27,5 +29,32 @@ def copy(
     )
 
 
+@app.command()
+def mass_single_column_update(
+    schema: Annotated[str, typer.Argument()],
+    target_column: Annotated[str, typer.Argument()],
+    new_column: Annotated[str, typer.Argument()],
+    data_type: Annotated[str, typer.Argument()],
+) -> None:
+    db_schema = Schema(name=schema)
+    target_column = Column(name=target_column, data_type=data_type)
+    new_column = Column(name=new_column, data_type=data_type)
+    logging.getLogger().setLevel("DEBUG")
```

Comment:
bit of nitpicking here but this should be set using an env variable (also, we shouldn't be using the root logger but at least one specific to the package :) )

Reply:
I agree, should be either be a env or a flag for --verbose. Will fix it before merging on monday. 👍